tests/package.json

```diff
@@ -4,16 +4,16 @@
   "description": "<p>  <img src=\"./images/open-data-discovery-platform-odd-logo.png\" width=\"600px\" alt=\"open-data-discovery-logo\"/>&nbsp; </p>",
   "main": "index.js",
   "directories": {
-    "doc": "docs"
+    "doc": "../docs"
   },
   "scripts": {
     "build": "tsc",
-    "clean": "rimraf dist/",
+    "clean": "./node_modules/.bin/rimraf ./dist/",
```

Comment:
probably we can use `npx rimraf ./dist/` instead of `./node_modules/.bin/rimraf ./dist/`

Reply:
Yep, even "rimraf ./dist/" be good